src/ng-select/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I think we should stick to using `npm` in here or move everything to `yarn`.

Reply:
Yeah ok, will remove the yarn.lock